Microsoft.Toolkit.Wpf.UI.Controls/MapControl/MapControl.cs

```diff
@@ -77,7 +77,6 @@ protected override void OnInitialized(EventArgs e)
             Bind(nameof(ZoomLevel), ZoomLevelProperty, windows.UI.Xaml.Controls.Maps.MapControl.ZoomLevelProperty);
             Bind(nameof(Center), CenterProperty, windows.UI.Xaml.Controls.Maps.MapControl.CenterProperty, new WindowsXamlHostWrapperConverter());
             Bind(nameof(LoadingStatus), LoadingStatusProperty, windows.UI.Xaml.Controls.Maps.MapControl.LoadingStatusProperty, new WindowsXamlHostWrapperConverter());
-            Bind(nameof(MapElements), MapElementsProperty, windows.UI.Xaml.Controls.Maps.MapControl.MapElementsProperty);
             Bind(nameof(Pitch), PitchProperty, windows.UI.Xaml.Controls.Maps.MapControl.PitchProperty);
             Bind(nameof(Routes), RoutesProperty, windows.UI.Xaml.Controls.Maps.MapControl.RoutesProperty);
             Bind(nameof(TileSources), TileSourcesProperty, windows.UI.Xaml.Controls.Maps.MapControl.TileSourcesProperty);
```

Comment:
Commit title says it also fixes TileSources, but I don't see changes in the PR for that?